doc/user-guide/pandas.rst

```diff
@@ -110,6 +110,62 @@ work even if not the hierarchical index is not a full tensor product:
     s[::2]
     s[::2].to_xarray()
 
+Lossless and reversible converter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The previous example shows that the conversion is not reversible (lossless roundtrip) and that the size of the ``datasets`` increases.
+
+Another approach is to use a lossless and reversible conversion (e.g Third party `ntv-pandas`__ libraries). A dataset can then be shared
+between several tools.
+
+__ https://github.com/loco-philippe/ntv-pandas/blob/main/README.md
+
+DataFrame to Dataset or DataArray
```

Comment:
It's not clear how you would transform to a DataArray

Reply:
I propose to add a Note: 

"a `dataset=True` or `dataset=False` parameter is used the choose the conversion result (active only if the Dataframe contains a single variable)"